Dockerfile

```diff
@@ -66,97 +66,139 @@ RUN chmod 644 $PROJECTOR_DIR/ide/projector-server/lib/*
+    && chown -R ${PROJECTOR_USER_NAME} /home/${PROJECTOR_USER_NAME}/.ssh \
+    && touch /usr/local/share/bash_history \
+    && chown ${PROJECTOR_USER_NAME} /usr/local/share/bash_history
+
+# Install additional OS packages.
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -y install --no-install-recommends bash-completion vim \
+    && apt-get autoremove -y && apt-get clean -y && rm -rf /var/lib/apt/lists/* \
+# Install Trino CLI
+    && wget https://repo1.maven.org/maven2/io/trino/trino-cli/${TRINO_VERSION}/trino-cli-${TRINO_VERSION}-executable.jar -P /usr/local/bin \
+    && chmod +x /usr/local/bin/trino-cli-${TRINO_VERSION}-executable.jar \
+    && ln -s /usr/local/bin/trino-cli-${TRINO_VERSION}-executable.jar /usr/local/bin/trino \
+# clean apt to reduce image size:
+     && apt-get autoremove -y \
+     && apt-get clean -y \
+     && rm -rf /var/lib/apt/lists/* $PROJECTOR_DIR/library-scripts/
 
-# Setup for Trino environment and with AzDevOps_azpcontainer user
-ARG USERNAME=AzDevOps_azpcontainer
-ARG USER_UID=1001 
-ARG USER_GID=$USER_UID
+# Use the Maven cache from the host and persist Bash history
+RUN mkdir -p /usr/local/share/m2 \
+    && chown -R ${PROJECTOR_USER_UID}:${PROJECTOR_USER_GID} /usr/local/share/m2 \
+    && ln -s /usr/local/share/m2 /home/${PROJECTOR_USER_NAME}/.m2
 
-RUN  true \
+# Add custom CA certificates to Java trust
+RUN true \
 # Any command which returns non-zero exit code will cause this shell script to exit immediately:
     && set -e \
 # Activate debugging to show execution details: all commands will be printed before execution
-    && set -x \
-    # Try to create a user with UID '1001' inside the container.
-    && useradd -m -u 1001 AzDevOps_azpcontainer \
-    # Grant user 'AzDevOps_azpcontainer' SUDO privilege and allow it run any command without authentication.
-    && groupadd azure_pipelines_sudo \
-    && usermod -a -G azure_pipelines_sudo AzDevOps_azpcontainer \
-    && su -c "echo '%azure_pipelines_sudo ALL=(ALL:ALL) NOPASSWD:ALL' >> /etc/sudoers"
-    # Allow user 'AzDevOps_azpcontainer' run any docker command without SUDO.
-    && stat -c %g /var/run/docker.sock \
-    && cat /etc/group \
-    && groupadd -g 997 azure_pipelines_docker \
-    && usermod -a -G azure_pipelines_docker AzDevOps_azpcontainer \
+    &&  set -x \
+    && for cert in /usr/local/share/ca-certificates/*; do \
+        openssl x509 -outform der -in "$cert" -out /tmp/certificate.der; \
+        $PROJECTOR_DIR/ide/jbr/bin/keytool -import -alias "$cert" -keystore $PROJECTOR_DIR/ide/jbr/lib/security/cacerts -file /tmp/certificate.der -deststorepass changeit -noprompt; \
+    done \
+    && rm /tmp/certificate.der
 
+USER $PROJECTOR_USER_NAME:$PROJECTOR_USER_GID
+ENV HOME /home/$PROJECTOR_USER_NAME
```

Comment:
This should be automatic. I don't think this is required...

Reply:
It was in the devcontainer dockerfile so that's why I brought it over...

Reply:
I kept this one.